docs/lab/overview/introduction.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: "Introduction"
+---
+
+Vital's lab test API allows digital health companies to carry out at-home lab testing.
+Companies can use the API to order a variety of tests, using various sampling methods.
+
+We partner with CLIA and CAP certified labs across the United States that cover a variety of diagnostic tests. For example lipids panels, metabolic profiles, hormone tests and more.
+We collect these test samples via dried bloodspot, saliva or phlebotomy.
+
+### Features
+
+Our lab test API gives you an end-to-end experience, from ordering a test all the way to receiving results, all with one API call!
+
+What we offer:
+
+* Multiple [test modalities](/lab/overview/testing-modalities) through one unified API.
+* No Physician required - you can use your own physician if you have one, or we will automatically assign one from our network.
```

Comment:
50 State Physician network - ..... 

we can't say no physician required, legal nightmare lol